files-community/Files
# Fix: Fixed high contrast support in the Properties window

**Resolved / Related Issues**
   Closes #issue...

**Validation**
How did you test these changes?
   1. Enable high contrast mode in Windows
   2. Open the Properties window

**Screenshots (optional)**
Add screenshots here.
